moj-node/server/routes/content.js

```diff
@@ -4,6 +4,7 @@ const express = require('express');
 module.exports = function createContentRouter({
   hubContentService,
   logger,
+  reportingUrl,
```

Comment:
This is more of a app wide dependency

just add it as follows:

in `app.js` find `app.locals.envVars = {`

and add the `reportingUrl ` in there

that way it will be available app wide